packages/grafana-ui/src/components/Graph/Graph.story.tsx

```diff
@@ -95,23 +77,46 @@ const series: GraphSeriesXY[] = [
   },
 ];
 
-export const withTooltip = () => {
-  const { tooltipMode } = getKnobs();
-  return (
-    <Graph
-      height={300}
-      width={600}
-      series={series}
-      timeRange={{
+export default {
+  title: 'Visualizations/Graph',
+  component: Graph,
+  decorators: [withCenteredStory],
+  parameters: {
+    knobs: {
+      disable: true,
+    },
+  },
+  args: {
+    series: series,
+    height: 300,
+    width: 600,
+    timeRange: {
+      from: dateTime(1546372800000),
+      to: dateTime(1546380000000),
+      raw: {
         from: dateTime(1546372800000),
         to: dateTime(1546380000000),
-        raw: {
-          from: dateTime(1546372800000),
-          to: dateTime(1546380000000),
-        },
-      }}
-      timeZone="browser"
-    >
+      },
+    },
+    timeZone: 'browser',
+    tooltipMode: 'single',
+  },
+  argTypes: {
+    tooltipMode: { control: { type: 'radio', options: ['multi', 'single'] } },
+    timeZone: { control: { type: 'radio', options: ['browser', 'utc'] } },
+    width: { control: { type: 'range', min: 200, max: 800 } },
+    height: { control: { type: 'range', min: 200, max: 800 } },
+    lineWidth: { control: { type: 'range', min: 1, max: 10 } },
+    className: NOOP_CONTROL,
+    series: NOOP_CONTROL,
+    timeRange: NOOP_CONTROL,
+    ariaLabel: NOOP_CONTROL,
+  },
+};
+
+export const WithTooltip: Story = ({ tooltipMode, ...args }) => {
```

Comment:
```suggestion
export const WithTooltip: Story<GraphProps & { tooltipMode: TooltipMode }> = ({ tooltipMode, ...args }) => {
```
You need to type the story pops here.